stdlib/public/core/Integers.swift.gyb

```diff
@@ -2320,24 +2292,24 @@ ${unsafeOperationComment(x.operator)}
   }
 % end
 
-  @inline(__always)
+  @_transparent
```

Comment:
It was changed from transparent because of the "sil explosion" at compile time. Watch the memory used by the swiftc while building the standard library, and if it hits ~6Gb – that's this.